src/main/java/com/khutircraftubackend/services/registration/RegistrationSellerController.java

```diff
@@ -0,0 +1,34 @@
+package com.khutircraftubackend.services.registration;
+
+import com.khutircraftubackend.dto.SellerDTO;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/v1/api")
+@RequiredArgsConstructor
+public class RegistrationSellerController {
```

Comment:
we can have `SellerController` for all sorts of requests related to seller.

Reply:
Without separation into the registration and CRUD?